bundles/persistence/org.openhab.persistence.rrd4j/README.md

```diff
@@ -63,38 +66,57 @@ Sets the timeintervall(seconds) between consecutive readings.
 For the first archive (and maybe the only one) a steps-size of one should be used.
 This way a sample is taken after each step.
-In this special case the selection of the consolidation function is of no effect (a single reading is equal to the MAX, MIN, AVERAGE and LAST of this reading).
   
 ### Example
 
-So in the above examples, we have 480 boxes, which each represent the value of one minute (Step is set to 60s, Granularity = 1).
-If more than one value is added to the database within (step x step) seconds (and thus more than one value would be stored in one box), the "consolidation function" is used.
+So in the example shown below, we have 480 boxes, which each represent the value of one minute (Step is set to 60s, Granularity = 1).
+If more than one value is added to the database within (step x steps) seconds (and thus more than one value would be stored in one box), the "consolidation function" is used.
 openHAB uses AVERAGE as default for numeric values, so if you add 20 and 21 within one minute, 20.5 would be stored.
 480 minutes is 8 hours, so we have a 8h with the granularity of one minute.
 
-The same goes for the next archives, for larger time spans, the stored values are less "exact". However, usually you are not interested in the exact temperature for a selected minute two years ago.
+The next archive has 144 boxes, which each represent the value of ten minutes (Step is set to 60s, Granularity = 10).
+1440 minutes is 24 hours, so we have a full day with with the granularity of 10 minutes.
+
+The same goes for following archives, for larger time spans, the stored values are less "exact". 
+However, usually you are not interested in the exact values for a selected minute some time ago.
+
+Note that the period covered for each archive starts with the actual time backwards! 
+No value is stored beyond the timeframe covered by the largest archive!
 
 services/rrd4j.cfg:
 
 ```
-ctr5min.def=COUNTER,900,0,U,300
-ctr5min.archives=AVERAGE,0.5,1,365:AVERAGE,0.5,7,300
-ctr5min.items=Item1,Item2
+ctr24h.def=COUNTER,900,0,U,60
+ctr24h.archives=AVERAGE,0.5,1,480:AVERAGE,0.5,10,144
+ctr24h.items=Item1,Item2
+ctr7d.def=COUNTER,900,0,U,60
+ctr7d.archives=AVERAGE,0.5,1,480:AVERAGE,0.5,10,144:AVERAGE,0.5,60,672
+ctr7d.items=Item3,Item4
+```
+In case no rrd4j.cfg is created the following default configuration will be used for all items persisted (i.e. all items with an allocated strategy in the rrd4j.persist file).
+
+Default rrd4j.cfg
+
+```
+defaultNumeric.def=GAUGE,60,U,U,60
+defaultNumeric.archives=AVERAGE,0.5,1,480:AVERAGE,0.5,4,360:AVERAGE,0.5,14,644:AVERAGE,0.5,60,720:AVERAGE,0.5,720,730:AVERAGE,0.5,10080,520
 ```
+The Datasource type is GAUGE, the heartbeat is 60s, MIN and MAX are unlimited and the step size is 60s.
+The archives are (no of boxes / granularity [minutes]): 480 / 1 (8 hrs), 360 / 4 ( 24 hrs), 644 / 14 (6.26 days), 720 / 60 (30 days), 730 / 720 (365 days), 520 / 10080 (10 years).
```

Comment:
I think this would be easier to understand as a bulleted list.

Reply:
I tried to keep it as short as possible. I'll give the bulleted list a try ( the formatting is my personell problem with that).